### PR TITLE
chore: removed -a shortcut from artifacts commands

### LINKED ISSUE #1062: Remove shortcuts from artifacts commands
We need to remove -a from +5 commands

_Originally posted by @wtrocki in https://github.com/redhat-developer/app-services-cli/pull/1054#discussion_r706049719_

### PR DESCRIPTION
**Pull request by alexal**
Closes #1062

### CODE REVIEW

File: pkg/cmd/registry/artifact/crud/create/create.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", opts.localizer.MustLocalize("registry.cmd.flag.output.description"))
 	cmd.Flags().StringVar(&opts.file, "file", "", opts.localizer.MustLocalize("artifact.common.file.location"))
 
-	cmd.Flags().StringVarP(&opts.artifact, "artifact-id", "a", "", opts.localizer.MustLocalize("artifact.common.id"))
+	cmd.Flags().StringVarP(&opts.artifact, "artifact-id", "", "", opts.localizer.MustLocalize("artifact.common.id"))
```
Comment (wtrocki) [inline pkg/cmd/registry/artifact/crud/create/create.go:103]: Doesn't need to be `StringVarP` anymore :)

**Comment by wtrocki** [comment]
@alexal You are unreal!

**Comment by wtrocki** [comment]
I have merged my command that does it so rebase is needed and status command also needs to be covered.

**Comment by alexal** [comment]
> I have merged my command that does it so rebase is needed and status command also needs to be covered.

Done, please take a look.

**Comment by wtrocki** [comment]
Docs weren't updated but will do that in next PR

**Pull request merged**